backend/data/models/leagueModel.js

```diff
@@ -0,0 +1,23 @@
+const db = require('../dbConfig.js');
+
+module.exports = {
+    getAll: (user) =>{
```

Comment:
isn't this going to return everything on the league table? There should be a way to only return all of the leagues that belong to a specific user.